src/main/java/javaserver/Methods/Delete.java

```diff
@@ -0,0 +1,17 @@
+package javaserver;
+
+import java.nio.file.Files;
+import java.io.IOException;
+
+public class Delete extends Handler {
```

Comment:
About the package naming: `Methods` should be lowercased since it is a package.

Also, maybe the package name should be `handlers` since these are all extending Handler